### PR TITLE
Add methods to get/update user information

### PR DESCRIPTION
**Pull request by re-ereit**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,6 +6,7 @@ require (
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/gin-gonic/gin v1.5.0
 	github.com/radovskyb/watcher v1.0.7
+	github.com/roboticeyes/gorexos v0.0.0-20200605091724-576326b6cdd1
```
Comment (breiting) [inline go.mod:9]: dependency to gorexos is not allowed by design!

File: rexos/user.go [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+	"github.com/tidwall/gjson"
+)
+
+// Address user address
+type Address struct {
+	Zip         *string `json:"zip,omitempty"  example:"8010"`
+	City        *string `json:"city,omitempty" example:"Graz"`
+	Address     *string `json:"address,omitempty" example:"Josef Huber Gasse"`
+	State       *string `json:"state,omitempty"`
+	CountryName *string `json:"countryname,omitempty" example:"Austria"`
```
Comment (breiting) [inline rexos/user.go:19]: countryName

File: rexos/user.go [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+		log.WithFields(event.Fields{
+			"status": ret,
+			"query":  query,
+		}).Error("Failed to get current user")
+
+		ret.Message = "Could not get current user. Please make sure you have the correct access rights."
+		return UserLicenses{}, ret
+	}
+
+	userLicensesLink := rexos.StripTemplateParameter(gjson.Get(string(currentUserResult), "_links.userLicenses.href").String())
```
Comment (breiting) [inline rexos/user.go]: remove `rexos.`

File: go.mod (hunk 1 above)
Comment (re-ereit) [reply go.mod]: removed

File: rexos/user.go (hunk 2 above)
Comment (re-ereit) [reply rexos/user.go:19]: this name is given from the backend

File: rexos/user.go (hunk 3 above)
Comment (re-ereit) [reply rexos/user.go]: removed

**Pull request merged**